android-fcm/README.md

```diff
@@ -30,9 +30,29 @@ Add the dependency to the Android project:
 
     compile "de.golfgl.gdxpushmessages:gdx-pushmessages-android-fcm:$pushMsgsVersion"
     
-    
-Note: Firebase Analytics is not included and Firebase Cloud Messanging's auto-init feature is disabled to respect the 
-privacy of your users.  
+
+
+| :information_source: Note | Firebase Analytics is not included and Firebase Cloud Messanging's auto-init feature is disabled to respect the privacy of your users. |
+| --- | --- |
+
+In order to change this behavior, modify your AndroidManifest.xml as shown below:
```

Comment:
I think it would be a bit clearer if the sentence would be 

> If you do want to use Firebase Analytics in your project, ...

Reply:
Done, I also mentioned Firebase Cloud Messanging's auto-init feature